develop/application-devs/interact/endpoints.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use. These endpoints can be used for development, RPC, or other purposes for which network access is required.
+---
+
+Ideally, one may utilize their own node when interacting with the [Polkadot network](https://polkadot.network/){target=_blank} via [Polkadot Js UI Apps](https://polkadot.js.org/apps/){target=_blank} or other user interfaces and programmatic methods. Another option would be to connect to one of the several public endpoints provided by infrastructure and API service providers. For development convenience, [Parity Tech](https://www.parity.io/){target=_blank} maintains archive nodes for Polkadot, Kusama, and their test networks with public endpoints. These endpoints can be used with [Polkadot Js UI API](https://polkadot.js.org/docs/api){target=_blank} to interact with their respective chains.
+
+### Network Endpoints
+
+Endpoints for all production and test networks are listed on the [Polkadot Js UI](https://polkadot.js.org/apps/#/accounts){target=_blank} which are accessed from [here](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints){target=_blank}. Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo test network are maintained by the community. System Chains as well as Westend and Rococo test network endpoints maintained by Parity Technologies are listed below (with the exception of [Paseo](https://github.com/paseo-network){target=_blank}, which is maintained by the wider community):
```

Comment:
It is not a good practice to use `here` for a link. Replace with a more self-descriptive word